tests/unit/parsers/test.manifestjson.js

```diff
@@ -394,6 +394,20 @@ describe('ManifestJSONParser', () => {
       expect(errors[0].code).toEqual(messages.MANIFEST_FIELD_INVALID.code);
       expect(errors[0].message).toContain('/name');
     });
+
+    it('should not contain the words "mozilla" or "firefox" on name value', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({ name: 'my-awesome-ext for Firefox' });
+      const manifestJSONParser = new ManifestJSONParser(
+        json,
+        addonLinter.collector
+      );
+      expect(manifestJSONParser.isValid).toEqual(false);
+      const { errors } = addonLinter.collector;
+      expect(errors[0].code).toEqual(
+        messages.PROP_NAME_MUST_NOT_CONTAIN_MOZILLA_OR_FIREFOX.code
+      );
+    });
```

Comment:
Can you also add a test that covers localized names?

Reply:
@wagnerand thanks for your review!

Before i start coding this i would like to share to you my solution proposal:

First i'll include a new `msgid` in all `messages.po` like:
(is there an automated way to do it? or it has to be manually?)

```
msgid "mozilla"
msgstr "<string for the equivalent value to the language of the country>"
```
```
msgid "firefox"
msgstr "<string for the equivalent value to the language of the country>"
```

Than i will use [this regex](https://github.com/mozilla/addons-linter/blob/e0e1946a5e88e3536386562c514f1f2a9afb8682/src/const.js#L215) to detect if the extension is localizing the extension name in the manifest.

If there is any localized character will loop through files and then check the msgid for the localized string. If match with `msgid "firefox"` or `msgid "mozilla"` will show a warning about invalid extension name.

is this the way?

I thought about this this solution from some tips from @rpl and @freaktechnik on the Add-on matrix channel.

Reply:
I don't think this is what you want. Have a look at https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization to see how localization works in extensions. Also `src/parsers/locale-messagesjson.js´ might be useful, it deals with localization.

Reply:
Thanks @wagnerand for the explanation! Now I understand the problem and update my PR with a solution with i've included a test to validate it too.